script.js

```diff
@@ -0,0 +1,8 @@
+var elements = document.querySelectorAll(".container__content-item");
```

Comment:
```suggestion
const elements = document.querySelectorAll(".container__content-item");
```

Remember to update this selector when you do the change about the item class